spring-boot-examples/README.md

```diff
@@ -0,0 +1,129 @@
+# Dapr Spring Boot and Testcontainers integration Example
+
+This example consist on two applications:
+- Producer App: 
+  - Publish messages using a Spring Messaging approach 
+  - Store and retrieve information using Spring Data CrudRepository 
+  - Implements a Workflow with Dapr Workflows
+- Consumer App:
+  - Subscribe to messages
+
+## Running this examples from source code
+
+To run this examples you will need: 
+- Java SDK
+- Maven
+- Docker or a container runtime such as Podman
+
+From the `spring-boot-examples/` directory you can start each service using the test configuration that uses 
+[Testcontainers](https://testcontainers.com) to boostrap [Dapr](https://dapr.io) by running the following command: 
+
+```bash
+cd producer-app/
+mvn spring-boot:test-run
+```
+
+This will start the `producer-app` with Dapr services and the infrastructure needed by the application to run, 
+in this case RabbitMQ and PostgreSQL. The `producer-app` starts in port `8080` by default.
+
+To run the `consumer-app` alongside the `producer-app` you need to change the test configuration slightly so both applications join the same 
+container network. Check the `DaprTestContainersConfig` class inside the 
+`test/java/io/dapr/springboot/examples/consumer` directory you need to uncomment the following line: 
+
+```java
+  @Bean
+  public RabbitMQContainer rabbitMQContainer(Network daprNetwork) {
+    return new RabbitMQContainer(DockerImageName.parse("rabbitmq:3.7.25-management-alpine"))
+            .withExposedPorts(5672).withNetworkAliases("rabbitmq")
+            //Uncomment to run this app alongside `producer-app` 
+            //.withReuse(true)
+            .withNetwork(daprNetwork);
+  }
+```
+
+`.withReuse(true)` needs to be uncommented for the `consumer-app` to connect to the same RabbitMQ broker 
+than the `producer-app`.
+
+You also need to make sure that the Dapr sidecar can connect to the same placement service started by the `producer-app`.
+Uncomment the following line: 
+
+```java
+ return new DaprContainer("daprio/daprd:1.14.4")
+            .withAppName("consumer-app")
+            .withNetwork(daprNetwork).withComponent(new Component("pubsub",
```

Comment:
Shouldn't we add the app health check here?

Reply:
done